app/models/slide.rb

```diff
@@ -107,8 +110,38 @@ def take_screenshot
     end
   end
 
+  def file_size
```

Comment:
This method name could be better.

It also might be good to split this method into two methods.  One for the foreground and one for the background.  They might even share some code, in which case you could use a private helper method that does the real work.